python/bench/io_compression.py

```diff
@@ -1,8 +1,29 @@
 import orjson
 import zlib
+import uuid
 import zstandard as zstd
 import time
-from typing import List, Tuple, Dict
+import io
+from typing import List, Dict, Tuple, Any
+from requests_toolbelt.multipart import encoder as rqtb_multipart
+
+
+class MultipartPartsAndContext:
+    def __init__(self, parts: List[Tuple[str, Tuple[None, bytes, str, Dict[str, str]]]], 
+                 context: str):
+        self.parts = parts
+        self.context = context
+
+class SerializedRunOperation:
+    def __init__(self, data: Dict[str, Any]):
+        self.operation = data["operation"]
+        self.id = data["id"]
+        self._none = orjson.dumps(data)
```

Comment:
you need to pop all of the other stuff before serializing this otherwise there is no point